model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

```diff
@@ -0,0 +1,629 @@
+    "\n",
+    "from monai.config import print_config\n",
+    "\n",
+    "print_config()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Ub0GzFRXemJN"
+   },
+   "outputs": [],
+   "source": [
+    "# If running on SageMaker or Studio Lab, install essential packages and extensions.\n",
+    "if \"studio-lab-user\" in os.getcwd():\n",
+    "    print(\"Upgrading dependencies\")\n",
+    "    !conda install --yes -q --prefix {sys.prefix} -c conda-forge opencv nodejs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BctRdoEcsf-G"
+   },
+   "source": [
+    "### On many systems you must manually install the imjoy-jupyter-extension for itkWidgets!\n",
+    "\n",
+    "If you do not see a blue 'ImJoy' icon on the menu bar in this notebook:\n",
+    "   1) Google CoLab: The following does not apply to Google CoLab - it will not show an ImJoy and all\n",
+    "      should work without modification.\n",
+    "   2) Enable Extensions:  Many Jupyter Lab systems disable jupyter extensions by default,\n",
+    "      and they must be enabled for this notebook to work. Typically this is accomplished using the \n",
+    "      Jupyter interface to select the extension manager (left-hand side, icon that looks like a piece\n",
+    "      of a puzzle) and select the Enable button if it appears.\n",
+    "   2) Install imjoy extension: In the extension manager, search for 'imjoy' and install\n",
+    "      the 'imjoy-jupyter-extension'. The installation can take several minutes. It may also prompt\n",
+    "      you to rebuild, save, and reload your jupyter environment as part of this process.  In the end,\n",
+    "      you should see a blue 'ImJoy' icon on the left side of the menu bar in this notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "di4lxyikemJN",
+    "outputId": "408c7a4d-492f-43cf-cb38-afa92d1f53d3"
+   },
+   "source": [
+    "# 2. Download the data from TCIA\n",
+    "\n",
+    "[Browsing Collections](https://www.cancerimagingarchive.net/collections) and viewing [Analysis Results](https://www.cancerimagingarchive.net/tcia-analysis-results/) of datasets on TCIA are the easiest ways to become familiar with what is available. These pages will help you quickly identify datasets of interest, find valuable, supporting data that are not available via our APIs (e.g. clinical spreadsheets, non-DICOM segmentation data), and answer the most common questions you might have about the datasets.  In this tutorial we'll be working with the [SPIE-AAPM-NCI PROSTATEx Challenges (PROSTATEx) collection](https://doi.org/10.7937/K9TCIA.2017.MURS5CL10.7937/TCIA.X0H0-1706), but if you browse the prevoiusly mentioned links you will find that TCIA has many other prostate datasets which could be used with [Prostate mri anatomy](https://github.com/Project-MONAI/model-zoo/releases/download/hosting_storage_v1/prostate_mri_anatomy_v0.3.1.zip) model. \n",
```

Comment:
Thank you @kirbyju for preparing the PR! Below are my comments:
Typo:"prevoiusly"

Reply:
The link ("https://doi.org/10.7937/K9TCIA.2017.MURS5CL10.7937/TCIA.X0H0-1706") does not work for me. Can you please check? Thanks!

Reply:
Good catches, thanks.